### ISSUE TITLE
TAGS=libvirt hack/build.sh in the installer directory bin file is not comming 

### ISSUE THREAD

**Issue by mayukh1** [openshift/installer#4217]
# Version

```console
$ openshift-install version 
<your output here>
``` 4.5

# Platform:
<!-- 
Please specify the platform type: aws, libvirt, openstack or baremetal
-->
Libvirt with KVM
<!--
Please specify:
* IPI (automated install with `openshift-install`. If you don't know, then it's IPI)
* UPI (semi-manual installation on customised infrastructure)
-->
IPI

# What happened?

Enter text here.
go: github.com/openshift/library-go@v0.0.0-20200324092245-db2a8546af81 requires
        bitbucket.org/ww/goautoneg@v0.0.0-20120707110453-75cd24fc2f2c: reading https://api.bitbucket.org/2.0/repositories/ww/goautoneg?fields=scm: 404 Not Found

# What you expected to happen?

This will create bin/openshift-install. This binary can then be invoked to create an OpenShift cluster, like so:
bin/openshift-install create cluster
The installer will show a series of prompts for user-specific information and use reasonable defaults for everything else. In non-interactive contexts, prompts can be bypassed by providing an install-config.yaml.

# How to reproduce it (as minimally and precisely as possible)?

<!--
Please list the full steps required to reproduce the issue.
-->

```console
$ your-commands-here
```

# Anything else we need to know?

Enter text here.

# References

- enter text here.

**Comment by cfergeau** [comment]
See https://github.com/openshift/installer/issues/4162 which is fixed now.

**Comment by ElCoyote27** [comment]
this was fixed for 4.6  but not 4.5, can we please get a fix in 4.5?

**Comment by ElCoyote27** [comment]
@cfergeau

**Comment by ElCoyote27** [comment]
Re-opening for release-4.5.
OCP 4.5 is still widely used (and is the only version currently supported by 'OpenStack Service Telemetry Framework' so please fix it for OCP 4.5 as well.

**Comment by staebler** [comment]
This will not be fixed in 4.5. It involves updating quite a few dependencies, which carries too much risk to backport, unfortunately.
